### PR TITLE
Teamdashboard template

### PR DESCRIPTION
**Pull request by nireeshT**
This PR include changes for 
Creating base template which can be extended by Capone/custom templates

### CODE REVIEW

File: src/app/screen_modules/team-dashboard/team-dashboard-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
+import {DashboardViewComponent} from './dashboard-view/dashboard-view.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: DashboardViewComponent,
+    children: [
+      { path: 'capone', component: DashboardViewComponent }
```
Comment (Sbrenthughes) [inline src/app/screen_modules/team-dashboard/team-dashboard-routing.module.ts]: Why are we using capone in the path?

Comment (Sbrenthughes) [reply src/app/screen_modules/team-dashboard/team-dashboard-routing.module.ts]: @nireeshT

Comment (nireeshT) [reply src/app/screen_modules/team-dashboard/team-dashboard-routing.module.ts]: @Sbrenthughes  done

File: src/app/shared/interfaces.ts [hunk 2]
```diff
@@ -22,3 +22,8 @@ export interface Chart {
     yAxisLabel: string;
     colorScheme: any;
 }
+
+export interface Widget {
```
[the remark this answers is not in the document]
Comment (nireeshT) [reply src/app/shared/interfaces.ts]: @Sbrenthughes  done

**Pull request merged**